airflow/providers/google/marketing_platform/example_dags/example_campaign_manager.py

```diff
@@ -48,6 +51,33 @@
     },
 }
 
+CONVERSION = {
+    "kind": "dfareporting#conversion",
+    "floodlightActivityId": 4327170,
```

Comment:
The following fields should be read from environment variables as they can be considered sensitive information and should not be hardcoded:
- floodlightActivityId
- floodlightConfigurationId
- mobileDeviceId